prov/efa/src/rxr/rxr_atomic.c

```diff
@@ -383,6 +380,8 @@ rxr_atomic_readwritemsg(struct fid_ep *ep,
 
 	ofi_ioc_to_iov(resultv, atomic_ex.resp_iov, result_count, datatype_size);
 	atomic_ex.resp_iov_count = result_count;
+	atomic_ex.result_desc = *result_desc;
```

Comment:
`result_desc` is an array, and can have multiple entry. number of entry is in `recult_ioc`.